src/test/scala/Tests.scala

```diff
@@ -1,176 +1,260 @@
-import org.scalatest._
 import java.io.File
 import org.scalatest.flatspec.AnyFlatSpec
-import matchers._
 import com.fasterxml.jackson.core.`type`.TypeReference
-import javax.lang.model.`type`.ErrorType
 import java.util.Calendar
 import java.util.GregorianCalendar
 
-class PersonalGamificationSpec extends AnyFlatSpec with should.Matchers:
-
-  it should "check to see if the add command is using the correct format" in {
-
-    val strSeq: Seq[String] = Seq("add", "--name=hello", "--effort=10")
-    val strSeq1: Seq[String] = Seq(
-      "add",
-      "name=hello, description=world, effort=12, due=2022-3-22"
-    )
-    val strSeq2: Seq[String] = Seq(
-      "add",
-      "--name=world",
-      "--effort=12",
-      "--description=hello my name is Justin",
-      "--due=2022-3-22"
-    )
-    val strSeq3: Seq[String] =
-      Seq("add", "aaaaaadkajkdjkajdkajdkajkdjkajdka111")
-    val strSeq4: Seq[String] =
-      Seq("add", "namehellodescriptionworldeffort122022-3-22")
-
-    //Correct format since --description, and --due are optional
-    Parser.apply(strSeq)
-    
-    //Error message should appear
-    Parser.apply(strSeq1)
-
-    //Correct format using --description and --due
-    Parser.apply(strSeq2)
-
-    //Error message should appear
-    Parser.apply(strSeq3)
-
-    //Error message should Appear
-    Parser.apply(strSeq4)
-  }
-
-  it should "check to see if the delete command is using the correct format" in {
-
-    val strSeq1: Seq[String] = Seq(
-      "--delete",
-      "--id=1"
-    )
+class PersonalGamificationSpec extends AnyFlatSpec:
 
-    val strSeq2: Seq[String] = Seq(
-      "delete",
-      "id=1"
-    )
+  val cal = Calendar.getInstance()
```

Comment:
All lifetimes are the same. This means that anything testing whether or not things have expired will all have the same result.